content/webapp/pages/stories/index.tsx

```diff
@@ -96,16 +96,20 @@ export const getServerSideProps: GetServerSideProps<
   // way we can be confident each of the three series that we have contains at
   // least one comic.
   const comics = transformQuery(comicsQuery, transformArticle);
+  const comicSeries = new Map<string, Series>();
+  for (const comic of comics.results) {
+    const series = comic.series[0];
```

Comment:
Should it be `comic.series?.[0]`?

Reply:
Nope - series is always defined, it just might be empty (the error was from `id` not being a property of `series[0]`)